### ISSUE TITLE
How do you generate the ground truth mask of TSR and TSSR?

### ISSUE THREAD

**Issue by shallweiwei** [naver/garnet#3]
Thanks for your work!
Is the mask of TSR generated by using Igt - I in ?
And could you release your training code?

**Comment by hsl323** [comment]
> Is the mask of TSR generated by using Igt - I in ?

The answer to your question is "Yes"
TSR mask can be generated by subtracting $I_{gt}$ from $I_{in}$.
As mentioned in the paper, we used 25 as the threshold to suppress the noise

$Mask_{TSR} = (I_{in} - I_{gt}) > 25 $

> And could you release your training code?

Due to the company's regulations, we have no plans to release the training code yet.
Please understand that we can't release it, and we apologize for the inconvenience.

**Comment by weinixuehao** [comment]
> > Is the mask of TSR generated by using Igt - I in ?
> 
> The answer to your question is "Yes" TSR mask can be generated by subtracting Igt from Iin. As mentioned in the paper, we used 25 as the threshold to suppress the noise
> 
> MaskTSR=(Iin−Igt)>25
> 
> > And could you release your training code?
> 
> Due to the company's regulations, we have no plans to release the training code yet. Please understand that we can't release it, and we apologize for the inconvenience.

![image](https://github.com/naver/garnet/assets/17869361/7a2e804c-05ea-4bfe-8ea2-9406c00a73b2)
Why my result is like this
